src/components/loader/loader.jsx

```diff
@@ -4,9 +4,11 @@ export function Loader() {
   const centerLoader = {
     display: "flex",
     height: "70vh",
-    width: "100vw",
+    width: "95vw",
     alignItems: "center",
     justifyContent: "center",
+    position: "absolute",
+    top: "0",
```

Comment:
Shouldn’t be string  top:0.

Reply:
both works, I chose to store them in string